src/runtime/graph_executor/graph_executor.cc

```diff
@@ -502,6 +502,14 @@ PackedFunc GraphExecutor::GetFunction(const std::string& name,
       dmlc::MemoryStringStream strm(const_cast<std::string*>(&param_blob));
       this->ShareParams(dynamic_cast<const GraphExecutor&>(*module.operator->()), &strm);
     });
+  } else if (name == "get_input_index") {
+    return PackedFunc([sptr_to_self, this](TVMArgs args, TVMRetValue* rv) {
+      if (String::CanConvertFrom(args[0])) {
+        *rv = this->GetInputIndex(args[0].operator String());
+      } else {
+        *rv = args[0];
+      }
```

Comment:
```suggestion
      CHECK(String::CanConvertFrom(args[0])) << "Input key is not a string";
      *rv = this->GetInputIndex(args[0].operator String());
```

IIUC, it means we simply return the input when it is not a string? It seems a bit weird.
`get_input` uses the similar logic because it allows you to get the input tensor using either name or index, but it is not the case here.